src/Logging/Logging.Console/src/ConsoleLoggerProvider.cs

```diff
@@ -3,197 +3,80 @@
-using System.Collections.Generic;
-using Microsoft.Extensions.Logging.Console.Internal;
+using System.Runtime.InteropServices;
+using Microsoft.Extensions.Logging.Abstractions;
 using Microsoft.Extensions.Options;
 
 namespace Microsoft.Extensions.Logging.Console
 {
-// IConsoleLoggerSettings is obsolete
-#pragma warning disable CS0618 // Type or member is obsolete
-
     [ProviderAlias("Console")]
     public class ConsoleLoggerProvider : ILoggerProvider, ISupportExternalScope
     {
-        private readonly ConcurrentDictionary<string, ConsoleLogger> _loggers = new ConcurrentDictionary<string, ConsoleLogger>();
-
-        private readonly Func<string, LogLevel, bool> _filter;
-        private IConsoleLoggerSettings _settings;
-        private readonly ConsoleLoggerProcessor _messageQueue = new ConsoleLoggerProcessor();
+        private readonly IOptionsMonitor<ConsoleLoggerOptions> _options;
+        private readonly ConcurrentDictionary<string, ConsoleLogger> _loggers;
+        private readonly ConsoleLoggerProcessor _messageQueue;
 
-        private static readonly Func<string, LogLevel, bool> trueFilter = (cat, level) => true;
-        private static readonly Func<string, LogLevel, bool> falseFilter = (cat, level) => false;
         private IDisposable _optionsReloadToken;
-        private bool _includeScopes;
-        private bool _disableColors;
-        private IExternalScopeProvider _scopeProvider;
-        private string _timestampFormat;
-        private LogLevel _logToStandardErrorThreshold;
-
-        [Obsolete("This method is obsolete and will be removed in a future version. The recommended alternative is using LoggerFactory to configure filtering and ConsoleLoggerOptions to configure logging options.")]
-        public ConsoleLoggerProvider(Func<string, LogLevel, bool> filter, bool includeScopes)
-            : this(filter, includeScopes, false)
-        {
-        }
-
-        [Obsolete("This method is obsolete and will be removed in a future version. The recommended alternative is using LoggerFactory to configure filtering and ConsoleLoggerOptions to configure logging options.")]
-        public ConsoleLoggerProvider(Func<string, LogLevel, bool> filter, bool includeScopes, bool disableColors)
-        {
-            if (filter == null)
-            {
-                throw new ArgumentNullException(nameof(filter));
-            }
-
-            _filter = filter;
-            _includeScopes = includeScopes;
-            _disableColors = disableColors;
-        }
+        private IExternalScopeProvider _scopeProvider = NullExternalScopeProvider.Instance;
 
         public ConsoleLoggerProvider(IOptionsMonitor<ConsoleLoggerOptions> options)
         {
-            // Filter would be applied on LoggerFactory level
-            _filter = trueFilter;
+            _options = options;
+            _loggers = new ConcurrentDictionary<string, ConsoleLogger>();
             _optionsReloadToken = options.OnChange(ReloadLoggerOptions);
```

Comment:
nit: I think this can be removed since it's registered in the `ReloadLoggerOptions` call we do immediately after